srcpkgs/lookbook/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'lookbook'
+pkgname=lookbook
+version=1.1.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config vala"
```

Comment:
Probably + `glib-devel` or just in case `if [ "$CROSS_BUILD" ]; then hostmakedepends+=" glib-devel"; fi`
Should fix cross builds.